### PR TITLE
RDAPI-14603 Add Security response headers section

### PR DESCRIPTION
**Pull request by sgoka-axway**
Thank you for your contribution to the Axway-Open-Docs repo.

## Describe the changes

Adding security response headers.

## Deploy preview link

https://deploy-preview-1840--axway-open-docs.netlify.app/docs/apimgmt_security/c_secgd_config_secconfig/

https://deploy-preview-1840--axway-open-docs.netlify.app/docs/apim_installation/apiportal_install/secure_harden_portal/#define-a-restrictive-content-security-policy

## Checklist for contributors

Before submitting this PR, please make sure:

* [x] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x] You have verified the technical accuracy of your change
* [x] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [x] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)
* [x] You have verified that all status checks have passed

_Put an x in the boxes that apply. This is simply a reminder of what we are going to look for before merging your change._

### CODE REVIEW

**Review by Rob-Conan — APPROVED** [review]
This looks good to me.

**Comment by andreamussap** [comment]
@sgoka-axway please see @mvenzke-axway comment on #1835

**Comment by kmercierwhite** [comment]
The changes look great but the grammar/formatting seems a bit inconsistent. If it's not an issue for Andrea, I'm happy to give my blessing.

The information is all very good from a technical perspective.

**Review by kmercierwhite — APPROVED** [review]

File: content/en/docs/apimgmt_security/c_secgd_config_secconfig.md [hunk 1]
```diff
@@ -6,6 +6,75 @@
...
+
+### X-XSS-Protection
+
+The HTTP X-XSS-Protection response header stops pages from loading when they detect reflected cross-site scripting (XSS) attacks.
+
+```
+Syntax: X-XSS-Protection: 1; mode=block
+```
+
+### NoSNiff
```
Comment (andreamussap) [inline content/en/docs/apimgmt_security/c_secgd_config_secconfig.md:50]: @Rob-Conan I've done some rewrite in this paragraph, but I'm not sure if I got everything right, so please review and let me know.

File: content/en/docs/apimgmt_security/c_secgd_config_secconfig.md [hunk 2]
```diff
@@ -6,6 +6,75 @@
...
+* Strict-Transport-Security
+* Content-Security-Policy
+* X-XSS-Protection
+* X-Content-Type-Options (NoSNiff)
+* X-Frame-Options
+* Cache-Control (no-cache)
+
+### HTTP Strict Transport Security
+
+A Strict-Transport-Security HTTP header is sent with each HTTPS response.
```
[the remark this answers is not in the document]
Comment (Rob-Conan) [reply content/en/docs/apimgmt_security/c_secgd_config_secconfig.md:22]: What is written here is correct. The policy is the overall security policy but "A Strict-Transport-Security HTTP header is sent with each HTTPS response" is correct.

File: content/en/docs/apimgmt_security/c_secgd_config_secconfig.md [hunk 3]
```diff
@@ -6,6 +6,75 @@
...
+Syntax: Strict-Transport-Security: max-age=<seconds>[; includeSubDomains]
+```
+
+The `max-age` parameter sets the time frame, in seconds, for the secure connection (HTTPS) requirement. It's recommended to set this to a high value, for example, several months. A value below `7776000` is considered too low. The `includeSubDomains` flag defines that the policy applies also for subdomains of the sender of the response.
+
+For more information, see [Configure HTTP Strict Transport Security](/docs/apim_policydev/apigw_gw_instances/configure_http_strict_transport_security).
+
+### Content Security Policy
+
+The Content-Security-Policy (CSP) header sets a policy that instructs the browser to only fetch resources, such as scripts, images, or objects, from the specified locations. After CSP is configured, a compliant browser loads any resources from locations not listed in the policy. The CSP header reduces an attacker’s ability to inject malicious content and helps to protect a web page from attacks like Cross-Site Scripting (XSS), dynamic code execution, and clickjacking.
```
Comment (Rob-Conan) [inline content/en/docs/apimgmt_security/c_secgd_config_secconfig.md:34]: "After CSP is configured, a compliant browser loads any resources from locations not listed in the policy". Remove the "any" and "not" in this sentence. 
-> "After CSP is configured, a compliant browser loads resources from locations listed in the policy"

File: content/en/docs/apimgmt_security/c_secgd_config_secconfig.md (hunk 1 above)
Comment (Rob-Conan) [reply content/en/docs/apimgmt_security/c_secgd_config_secconfig.md:50]: "NoSNiff" -> No-Sniff. Same for the title above. 
"header protects web" -> "header **helps** protects web"
"Safari browsers treat responses" -> "Safari browsers **may** treat responses"

**Pull request merged**